packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -2609,6 +2619,15 @@ function flushPassiveEffectsImpl() {
     markPassiveEffectsStopped();
   }
 
+  if (__DEV__ && enableDoubleInvokingEffects) {
+    commitDoubleInvokeEffectsInDEV(root.current, true);
+  }
+
+  if (enableSchedulerTracing) {
+    popInteractions(((prevInteractions: any): Set<Interaction>));
+    finishPendingInteractions(root, lanes);
+  }
```

Comment:
I just noticed one more small thing.

I wonder if we should move this block:
```js
  if (__DEV__) {
    isFlushingPassiveEffects = false;
  }
```

Down until after we call `commitDoubleInvokeEffectsInDEV` ?

I don't think it *actually* matters, since that flag only controls whether we warn about state updates from unmounted components *but* in case it ever does more than that– might be good to keep the double-invoked effects inside of that toggle too :grin: